attest/core/src/types/report_body.rs

```diff
@@ -164,9 +165,9 @@ impl ReportBody {
 
         // Check if the security version is high enough
         let svn = self.security_version();
-        if minimum_security_version > svn {
+        if minimum_security_version.as_ref() > svn.as_ref() {
```

Comment:
The `as_ref()` is currently needed until https://github.com/mobilecoinfoundation/sgx/issues/374 gets done.